### PR TITLE
Added tests for iputils

### PR DESCRIPTION
**Pull request by Puthranps**
These tests reference to issue #1264 on prysm.
The purpose behind the changes is to validate the results of the return value for the function
ExternalIPv4. Using regular expressions to validate
the IP address. Bazel build configuration has also
been updated.

[Part of|Resolves] #1264 

---

# Description

To further validate the accuracy of ExternalIPv4's string value. 

Changes include the following:
  - Creating an external_ip_test.go file 
      * This file validates the return value by checking for error values and regex for the IP string. 
  - Changing the BUILD.bazel to add the newly written test file.

Feel free to send feedback, it would be much appreciated :smile:

### CODE REVIEW

File: shared/iputils/external_ip_test.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import (
+	"regexp"
+	"testing"
+)
+
+func TestExternalIPv4(t *testing.T) {
+	test, err := ExternalIPv4()
+
+	if err != nil {
+		t.Errorf("Test check external ipv4 failed with %g", err)
```
Comment (terenc3t) [inline shared/iputils/external_ip_test.go]: use %v to be consistent across code case

File: shared/iputils/external_ip_test.go [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	}
+
+	valid := regexp.MustCompile(`\.\d{1,3}\.\d{1,3}\b`)
+
+	if !valid.MatchString(test) {
+		t.Errorf("Test check check external ipv4, string does not match regexp")
```
Comment (terenc3t) [inline shared/iputils/external_ip_test.go]: can you use `wanted: but got:` pattern?

Comment (Puthranps) [reply shared/iputils/external_ip_test.go]: Sure, I'll make changes sometime tomorrow it's very late

**Comment by Puthranps** [comment]
Changes were made can you guys take a look?

**Pull request merged**